### PR TITLE
[github actions] ping actor of workflow on cherry pick pr creation

### PR DESCRIPTION
**Pull request by XilaiZhang**
Context: Follow up from sync with @itsjustkevin today. Tag and ping the actor of workflow, when the cherry pick pull request is successfully created.

1. actor of workflow will be the username of the user that triggered the initial workflow run. i.e., whoever labels the PR. This person might not necessarily be the author of the cherry pick PR.
2. PR number of newly created cherry pick PR is parsed from the output of "gh pr create" as a multi line string.

Tested: on cp success: https://github.com/flutter/flutter/pull/142674 pings the actor of workflow (not the pr author). on cp failure: [example workflow](https://github.com/XilaiZhang/miscellaneous-side-project/actions/runs/7734400373/job/21088324523) leaves a msg on the original PR.

### CODE REVIEW

File: .github/workflows/easy-cp.yml
Comment (CaseyHillers) [inline .github/workflows/easy-cp.yml:76]: Can we make this more concise by combing both lines?

eg `@actor please fill out the PR description above, afterwards the release team will review this request`

File: .github/workflows/easy-cp.yml
Comment (CaseyHillers) [inline .github/workflows/easy-cp.yml:78]: Do we need to generalize this so it supports the engine repo?

File: .github/workflows/easy-cp.yml
Comment (CaseyHillers) [inline .github/workflows/easy-cp.yml:63]: What does curly brackets mean?

File: .github/workflows/easy-cp.yml
Comment (CaseyHillers) [inline .github/workflows/easy-cp.yml:75]: Would it be easier to leave a comment on the original PR about the CP PR instead of the current approach where we leave a comment on the new PR?

File: .github/workflows/easy-cp.yml
Comment (XilaiZhang) [reply .github/workflows/easy-cp.yml]: Sure, updated wordings

File: .github/workflows/easy-cp.yml
Comment (XilaiZhang) [reply .github/workflows/easy-cp.yml:77]: Yeah I am planning to add a similar workflow in a separate PR to the engine repo after this one lands. It would trigger on labeling events of the engine repo.

(will also need to set up a secret on the engine repo)

File: .github/workflows/easy-cp.yml
Comment (XilaiZhang) [reply .github/workflows/easy-cp.yml:75]: Yeah my understanding is that commenting on the original PR or the new PR will have the same number of steps, because we want to provide the link of the new PR so that people could find it easily.

And to get the PR link of the new PR we can only parse the output from 'gh pr create'.

**Review by CaseyHillers — APPROVED** [review]

**Pull request merged**
